sys-libs/ncurses/ncurses6-6.1.recipe

```diff
@@ -0,0 +1,223 @@
+SUMMARY="New curses library"
+HOMEPAGE="http://www.gnu.org/software/ncurses/ncurses.html"
+SOURCE_URI="http://ftp.gnu.org/pub/gnu/ncurses/ncurses-$portVersion.tar.gz"
```

Comment:
Shall we switch to https and prepend the URI of the mirror redirector?
```diff
-SOURCE_URI="http://ftp.gnu.org/pub/gnu/ncurses/ncurses-$portVersion.tar.gz"
+SOURCE_URI="https://ftpmirror.gnu.org/ncurses/ncurses-$portVersion.tar.gz
+	https://ftp.gnu.org/gnu/ncurses/ncurses-$portVersion.tar.gz"
```
Notice: The `/gnu/` is not required for the first URI (i.e. the one with the `ftpmirror.gnu.org`) but it still required for the second URI (i.e. the one with the `ftp.gnu.org`).